examples/basic/index.ts

```diff
@@ -15,7 +15,8 @@ async function main() {
 				p.text({
 					message: 'Where should we create your project?',
 					placeholder: './sparkling-solid',
-					validate: (value) => {
+					validate: async (value) => {
+						await setTimeout(299);
```

Comment:
Curious as to why you've added the timeout here when the delay is below 300 ms? Meaning it won't show, right? Why not leave it out all together then?

Reply:
Ooh yeah that was just for testing purposes, can remove!